test.js

```diff
@@ -17,11 +17,11 @@ describe('repos', () => {
     expect(repos.express).to.equal('https://github.com/expressjs/express')
   })
 
-  it('is always a URL', function () {
-    this.timeout(10 * 1000)
+  it('is always a URL or null', function () {
+    this.timeout(30 * 1000)
     const urls = Object.values(repos)
     urls.forEach(url => {
-      expect(isUrl(url), `${url}`).to.eq(true)
+      expect(url === null || isUrl(url), `${url}`).to.eq(true)
```

Comment:
What about making sure that both URLs and nulls exist? Something like this:

```js
expect(urls.some(url => isUrl(url)).to.eq(true)
expect(urls.some(url => url === null).to.eq(true)
```

Reply:
good point, added it to test. Thanks.